.github/workflows/source_build.yml

```diff
@@ -11,28 +11,8 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-      - name: Set up JDK 11
-        uses: actions/setup-java@v1
-        with:
-          java-version: 11
-      - uses: actions/cache@v1
-        with:
-          path: ~/.m2/repository
-          key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
-          restore-keys: |
-            ${{ runner.os }}-maven-
-      - name: Install slice2java
-        run: |
-          sudo apt-get install -y libmcpp-dev
-          wget -q https://github.com/ome/zeroc-ice-ubuntu2004/releases/download/0.2.0/ice-3.6.5-0.2.0-ubuntu2004-amd64.tar.gz
-          tar xf ice-3.6.5-0.2.0-ubuntu2004-amd64.tar.gz
-          mv ice-3.6.5-0.2.0 ice-3.6.5
-          mv ice-3.6.5 /opt
-          rm ice-3.6.5-0.2.0-ubuntu2004-amd64.tar.gz
-          echo "/opt/ice-3.6.5/bin" >> $GITHUB_PATH
+      - name: Install Ice Java and Python binding
```

Comment:
FWIW, I could live without the name for the simplest of actions. Either way, :heart: for the reduction in lines!

Reply:
going for a minimalist style :-)